fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -1486,7 +1486,9 @@ private void beginTxn(String dbName, String tblName) throws UserException, TExce
             InterruptedException, ExecutionException, TimeoutException {
         TransactionEntry txnEntry = context.getTxnEntry();
         TTxnParams txnConf = txnEntry.getTxnConf();
-        long timeoutSecond = ConnectContext.get().getExecTimeout();
+        SessionVariable sessionVariable = ConnectContext.get().getSessionVariable();
+        int timeoutSecond = sessionVariable.getQueryTimeoutS();
```

Comment:
timeout should keep the original logic, because getExecTimeout may be query_timeout or insert_timeout

Reply:
resolved with a new commit, now I use `ConnectContext.get().getExecTimeout()` to handle timeout